hnn_core/gui/_viz_manager.py

```diff
@@ -252,8 +252,10 @@ def _update_ax(fig, ax, single_simulation, sim_name, plot_type, plot_config):
 
     elif plot_type == 'PSD':
         if len(dpls_copied) > 0:
+            min_f = plot_config['min_spectral_frequency']
+            max_f = plot_config['max_spectral_frequency']
             color = ax._get_lines.get_next_color()
-            dpls_copied[0].plot_psd(fmin=0, fmax=50, ax=ax, color=color,
+            dpls_copied[0].plot_psd(fmin=min_f, fmax=max_f, ax=ax, color=color,
```

Comment:
This is great! Can you checkout the spectrogram tests associated with #882 to verify that the PSD plot is rendered correctly?